backend/src/main/java/ch/nova_omnia/lernello/api/LearningKitRestController.java

```diff
@@ -62,8 +62,14 @@ public UUID delete(@PathVariable UUID learningKitId) {
     @PreAuthorize("hasAuthority('SCOPE_kits:read')")
     public @Valid List<LearningKitResDTO> getAll() {
         List<LearningKit> learningKits = learningKitService.findAll();
-        List<LearningKitResDTO> learningKitResDTOs = learningKits.stream().map(learningKitMapper::toDTO).collect(Collectors.toList());
-        return learningKitResDTOs;
+        return learningKits.stream().map(learningKitMapper::toDTO).collect(Collectors.toList());
+    }
+
+    @GetMapping("/latest-five")
+    @PreAuthorize("hasAuthority('SCOPE_kits:read')")
+    public @Valid List<LearningKitResDTO> getLatestFive() {
```

Comment:
Usually done via "Paginated" results, which Spring supports via the repositories already, but I'm okay with this as a quick implementation